tests/neg/i1212.scala

```diff
@@ -0,0 +1 @@
+@ann class ann extends scala.annotation.Annotation // error: cyclic reference
```

Comment:
I found out that the issue was closed before and piggy-backed on this one. Not according to ettiquette, I know. My only excuse is lack of time.